core/components/atoms/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -25,23 +25,50 @@ export interface BreadcrumbsProps extends BaseProps {
    * onClick handler for `Breadcrumb`
    */
   onClick?: (link: string) => void;
+  /**
+   * Determines whether to show tooltip for label
+   */
+  showTooltip?: boolean;
+}
+
+interface renderLinkProps {
+  item: Breadcrumb;
+  onClick?: (link: string) => void;
+  index?: number;
+  showTooltip?: boolean;
 }
 
-const renderLink = (item: Breadcrumb, onClick: BreadcrumbsProps['onClick']) => (
-  <Link
-    href={item.link}
-    onClick={(ev) => {
-      if (onClick) {
-        ev.preventDefault();
-        onClick(item.link);
-      }
-    }}
-    appearance="subtle"
-    size="tiny"
-  >
-    {item.label}
-  </Link>
-);
+const RenderLink = ({ item, onClick }: renderLinkProps) => {
+  const onClickHandler = (ev: React.MouseEvent) => {
+    if (onClick) {
+      ev.preventDefault();
+      onClick(item.link);
+    }
+  };
+
+  return (
+    <span className="Breadcrumbs-link ellipsis--noWrap" data-test="DesignSystem-Breadcrumbs-link">
+      <Link href={item.link} onClick={onClickHandler} appearance="subtle" size="tiny">
+        {item.label}
+      </Link>
+    </span>
+  );
+};
+
+const WithTooltip = ({ item, onClick, index, showTooltip }: renderLinkProps) => {
```

Comment:
This component should take only 2 props: `showTooltip` & `ReactElement` to render.